### PR TITLE
Add csv download buttons to bank transactions and pnl

### PR DESCRIPTION
**Pull request by AndrewAcomb**
fixes: LAY-721

Also removes the non-functional download button on the GL page

### CODE REVIEW

File: src/views/Reports/Reports.tsx [hunk 1]
```diff
@@ -13,28 +17,57 @@ export interface ReportsPanelProps {
...
+              year: year,
+              month: month,
+            },
+          },
+        )
+        try {
+          const result = await getProfitAndLossCsv()
+          if (result?.data?.presignedUrl)
+            window.location.href = result.data.presignedUrl
+        } catch (e) {}
```
Comment (doneel) [inline src/views/Reports/Reports.tsx:45]: Nervous about errors - lots of surface area for silent failure.  We have an error notification pattern, can you try seeing if there's anything useful for this case?

**Review by doneel — APPROVED** [review]
Request to look into showing an error has happened but your judgmental call on what's worth it atm.

**Comment by AndrewAcomb** [comment]
Screenshot 2024-06-06 at 6 29 18 PM

No longer silently failing

File: src/views/Reports/Reports.tsx (hunk 1 above)
Comment (AndrewAcomb) [reply src/views/Reports/Reports.tsx]: Used the failed / retry button pattern from the bank transaction categorization approval button

**Pull request merged**